ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -3,9 +3,13 @@
 import numpy as np
 
 class TLClassifier(object):
-    def __init__(self):
-        #TODO load classifier
-        pass
+    def __init__(self, is_site, classifier):
+        self.is_site = is_site
+        self.classifier = classifier
+        print "is_site ",self.is_site
```

Comment:
think we should standardize and all use rospy.loginfo / logdebug etc.

Reply:
Resolved